mitodl/mitxpro
# Added robots.txt via django-robots

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #644 

#### What's this PR do?

This PR introduces the `django-robots` app as a forward-thinking integration of needing separate robots.txt configurations for different environments. It also permits robots.txt to be changed at runtime, which is probably beneficial form a marketing perspective since it can be updated outside of our normal deployment cadence, or updated in a deployment via a migration.

- Removes legacy static robots.txt
- Adds `django-robots` app which allows us to configure robots.txt via django admin
- Provides a default set of configuration that disallows all robots on all urls
- Enables caching of robots.txt

#### How should this be manually tested?
- Start up the app and run the migrations, go to /robots.txt and verify you see this default:
```
User-agent: *
Disallow: /
```
- Go to `/admin/robots` and reconfigure things. You should see those changes reflected.
